src/App.tsx

```diff
@@ -15,71 +15,87 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  handleSelect = (goodText: string) => {
+    const goodTitle = goodText;
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+    if (this.state.selectedGood === goodText) {
+      this.setState({ selectedGood: '' });
+    } else {
+      this.setState({ selectedGood: goodTitle });
+    }
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  handleUnselect = () => {
+    this.setState({ selectedGood: '' });
+  };
```

Comment:
you no need separate function for selecting if you already do it in the handleSelect function

Reply:
not fixed